main/EDMainMod.F90

```diff
@@ -286,6 +287,11 @@ subroutine ed_integrate_state_variables(currentSite, bc_in )
                currentPatch%patchno,currentPatch%area
        endif
 
+       ! add age increment to secondary forest patches as well
+       if (currentPatch%anthro_disturbance_label .eq. secondaryforest) then
+          currentPatch%age_since_anthro_disturbance = currentPatch%age_since_anthro_disturbance + hlm_freq_day
```

Comment:
@ckoven , does age_since_anthro_disturbance tell us anything different than "age"?

Reply:
yes.  age gets reset whenever any disturbance happens, age_since_anthro_disturbance only gets reset during a harvest event.  So, if you have a secondary forest patch that also has other disturbance on it, this information allows you to do something like still go and harvest all the land on a set forest rotation schedule.

Reply:
So if a secondary forest patch experiences a tree-fall event, the patch that is created should not reset its age_since_antrho_disturbance, instead it retains that value.

Reply:
that's the idea.